st2api/st2api/controllers/v1/pack_config_schemas.py

```diff
@@ -24,17 +24,17 @@
 
 
 __all__ = [
-    'PackConfigSchemaController'
+    'PackConfigSchemasController'
 ]
 
 
-class PackConfigSchemaController(ResourceController):
+class PackConfigSchemasController(ResourceController):
```

Comment:
I've added "s" suffix to make it consistent with other top-level controllers. First it had no suffix since it lived as a nested controller under packs, but when moving it to a top-level I forgot to add "s" suffix.

All other top-level controllers use plural form, so we should also use it here for consistency. But yeah, I'm all for getting rid of it in API v2 where the path will probably look something like `/v2/packs/<pack ref>/config_schema`.